docs/src/pages/demos/breadcrumbs/RouterBreadcrumbs.js

```diff
@@ -56,75 +56,66 @@ const styles = theme => ({
   nested: {
     paddingLeft: theme.spacing(4),
   },
-});
+}));
 
 const LinkRouter = props => <Link {...props} component={RouterLink} />;
 
-class RouterBreadcrumbs extends React.Component {
-  state = {
-    open: true,
-  };
+function RouterBreadcrumbs() {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState(true);
 
-  handleClick = () => {
-    this.setState(state => ({ open: !state.open }));
-  };
-
-  render() {
-    const { classes } = this.props;
+  function handleClick() {
+    setOpen(!open);
```

Comment:
```suggestion
    setOpen(prevOpen =>!prevOpen);
```
This should work as well.

Reply:
There are other places in the code that can be changed to this too.

Reply:
Thanks :+1: